### ISSUE TITLE
Still no solution to "autorun" the controller script at boot?

### ISSUE THREAD

**Issue by TheBloodstomper** [csutorasa/XOutput#822]
Hey,

Running xOutput in a setup that is supposed to use no keyboard/mouse (arcade cabinet), I'd like to have my controller starting when booting the program, but `xoutput --start="controller name"` doesn't seem to work anymore, giving me a `"Unused command line argument: *****"`

Any new solution?

**Comment by Obvman** [comment]
Are you running version 3? In the controller edit window, there is a "Start emulation when controller is connected" option. It should connect the controller on application startup if it's ticked, does that work for you?

**Comment by TheBloodstomper** [comment]
> Are you running version 3? In the controller edit window, there is a "Start emulation when controller is connected" option. It should connect the controller on application startup if it's ticked, does that work for you?

Yes. It is ticked but doesn't seem to work. Maybe it's related to the fact that I start it at boot via an vbs script?
